sandy/sections/mf7.py

```diff
@@ -0,0 +1,448 @@
+        }
+        Num_beta = C.NBT[0]
+        add_beta = {}
+        for j in range(Num_beta):
+            add_temp = {}
+            T, i = sandy.read_tab1(df, i)
+            temp = T.C1  # Temperature
+            b = T.C2  # beta
+            LT = T.L1  # Temperature flag
+            add_2 = {
+                "LT": LT,
+                "INT": T.INT,
+                "NBT": T.NBT,  # Number of different pairs alpha S
+                "S": T.y,
+                }
+            add["alpha"] = T.x  # Alpha values, constant values
+            add_temp[temp] = add_2
+            for z in range(LT):
+                C, i = sandy.read_list(df, i)
+                temp = C.C1  # Temperature
+                add_2 = {
+                    "LI": C.L1,
+                    "S": C.B,
+                    }
+                add_temp[temp] = add_2
+            add_beta[b] = add_temp
+        add['beta/T'] = add_beta
+        add_efective_temp = {}
+        C, i = sandy.read_tab1(df, i)
+        add_2 = ({
+                "NR": C.NBT,
+                "NP": C.INT,
+                "TINT": C.x,
+                "T_eff": C.y,
+                })
+        add_efective_temp[0] = add_2
+        add['effective T'] = add_efective_temp
+        if NS > 1 and B[6] == 0:
+            C, i = sandy.read_tab1(df, i)
+            add_2 = ({
+                "NR": C.NBT,
+                "NP": C.INT,
+                "TINT": C.x,
+                "T_eff": C.y,
+                })
+            add['effective T'][1] = add_2
+            if NS > 2 and B[12] == 0:
+                C, i = sandy.read_tab1(df, i)
+                add_2 = ({
+                    "NR": C.NBT,
+                    "NP": C.INT,
+                    "TINT": C.x,
+                    "T_eff": C.y,
+                    })
+                add['effective T'][2] = add_2
+            if NS > 3 and B[18] == 0:
+                C, i = sandy.read_tab1(df, i)
+                add_2 = ({
+                    "NR": C.NBT,
+                    "NP": C.INT,
+                    "TINT": C.x,
+                    "T_eff": C.y,
+                    })
+                add['effective T'][3] = add_2
+        out.update(add)
+    return out
+
+
+def _write_elastic_scattering(sec):
+    """
+    Given the content of MF7 for elastic scattering as nested dictionaries,
+    write it to string.
+
+    Parameters
+    ----------
+    sec : 'dict'
+        Multiline string reproducing the content of a ENDF-6 section.
+
+    Returns
+    -------
+    `str`
+        Multiline string reproducing the content of a ENDF-6 section.
+
+    Examples
+    --------
+    Coherent elastic scattering:
+    >>> tls = sandy.get_endf6_file("endfb_80", 'tsl', 26)
+    >>> sec = sandy.sections.mf7._read_elastic_scattering(tls, 26, 2)
+    >>> assert len(_write_elastic_scattering(sec)) == len(tls.data[(26, 7, 2)])
+
+    This test is also check:
+    >>> #assert print(_write_elastic_scattering(sec)) == print(tls.data[(26, 7, 2)])
```

Comment:
either you keep it or you remove it

Reply:
I think that this example must be removed or changed. Even if you do `print(1) == print(3)` the result will be `True` because print returns _None_, indeed `print(1) == None`.